cdap-ui/app/features/mapreduce/controllers/tabs/runs/tabs/status-ctrl.js

```diff
@@ -27,22 +27,34 @@ angular.module(PKG.name + '.feature.mapreduce')
         runId: runid,
         scope: $scope
       };
-      myMapreduceApi.pollInfo(params)
```

Comment:
can we remove the myMapReduceApi import?
or remove the method defined on the API object?